content/rocur/about/index.en.md

```diff
@@ -29,7 +29,9 @@ Check out [@WeAreRLadies](https://twitter.com/WeAreRLadies) every Monday for a n
   
 ## Curator Schedule 
 
-**[current curator schedule](https://docs.google.com/spreadsheets/d/13NwIphQ6o-3YJUbHtbDRf4texfMOCvhIDNZgDZhHv7U/edit?usp=sharing)**  
+**[Nominate a curator](https://docs.google.com/forms/d/e/1FAIpQLSeGfdw2jbX6sqyC9PYMEWpr-RtBrtfUiqeZFfCUr9HhZmzqAQ/viewform)**  
```

Comment:
Maybe we can add the link after "Sign up to be a curator"?  

Then, we can add a section after the *Be A Curator!* section below (around line 54) to provide more explanation around nominations. What do you think about this?

Some potential text for this could be:

## Nominate a Curator!  
  
Do you know an R-Lady who you think would be a great curator? We are currently accepting nominations for curators via our [nomination form](https://docs.google.com/forms/d/e/1FAIpQLSeGfdw2jbX6sqyC9PYMEWpr-RtBrtfUiqeZFfCUr9HhZmzqAQ/viewform). 

Anyone is welcome to nominate curators - you do not need to be affiliated with R-Ladies to nominate an individual to curate for @WeAreRLadies. Multiple nominations from a single nominator is also permitted. @WeAreRLadies administrators will review nominations and invite nominees to curate.

Reply:
done